software/src/8.0/src/M_SunOS/make.CC

```diff
@@ -16,10 +16,10 @@ CDEFS	= -D_POSIX_C_SOURCE=199506L -D__EXTENSIONS__ -DVisionBuild
 CINCS	=
 CBASE	= -mt -KPIC -template=no%extdef
 CDBG	= -g -xs
-CREL	= -g0 -O5 -xvector
+CREL	= -O -xvector
 CVER	= ${CREL}
 C32	= -xarch=v8plus sun.v8plus.il
-C64	= -xarch=v9 -xmemalign=8i sun.v9.il
```

Comment:
-xarch=v9 still causes warning message even when paired with -m64
-xarch=sparc is allowed with no complaints, but I am not optimistic that it will help your situation. Still, it couldn't hurt to try